templates/focus-areas.md

```diff
@@ -0,0 +1,58 @@
+# Templates for focus-areas READMEs in WG repositories
+
+This  file is divided into 2 parts:
+1. Template for `WG/focus-areas/README.md`
+2. Template for `WG/focus-areas/<focus-area-name>/README.md`
+
+## Part 1
+
+The following template describes the structure of READMEs for focus-areas in each WG repositories:
+
+---
```

Comment:
Instead of lines, use 
\`\`\`

That will make the template look more like a template and can be copied more easily.

Reply:
Done